_posts/2020-05-30-Metal3-Cluster-Deployment.md

```diff
@@ -0,0 +1,544 @@
+```console
+kubectl get KubeadmControlPlane ${CLUSTER_NAME} -n metal3
+kubectl describe KubeadmControlPlane ${CLUSTER_NAME} -n metal3
+```
+
+```yaml
+Name:         eko
+Namespace:    metal3
+Labels:       cluster.x-k8s.io/cluster-name=eko
+Annotations:  <none>
+API Version:  controlplane.cluster.x-k8s.io/v1alpha3
+Kind:         KubeadmControlPlane
+Metadata:
+  Creation Timestamp:  2020-05-22T19:05:48Z
+  Finalizers:
+    kubeadm.controlplane.cluster.x-k8s.io
+  Generation:  1
+  Managed Fields:
+    API Version:  controlplane.cluster.x-k8s.io/v1alpha3
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:spec:
+        .:
+        f:infrastructureTemplate:
+          .:
+          f:apiVersion:
+          f:kind:
+          f:name:
+        f:kubeadmConfigSpec:
+          .:
+          f:files:
+          f:initConfiguration:
+            .:
+            f:nodeRegistration:
+              .:
+              f:kubeletExtraArgs:
+                .:
+                f:node-labels:
+              f:name:
+          f:joinConfiguration:
+            .:
+            f:controlPlane:
+            f:nodeRegistration:
+              .:
+              f:kubeletExtraArgs:
+                .:
+                f:node-labels:
+              f:name:
+          f:postKubeadmCommands:
+          f:preKubeadmCommands:
+          f:users:
+        f:replicas:
+        f:version:
+    Manager:      OpenAPI-Generator
+    Operation:    Update
+    Time:         2020-05-22T19:05:48Z
+    API Version:  controlplane.cluster.x-k8s.io/v1alpha3
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:finalizers:
+          .:
+          v:"kubeadm.controlplane.cluster.x-k8s.io":
+        f:labels:
+          .:
+          f:cluster.x-k8s.io/cluster-name:
+        f:ownerReferences:
+          .:
+          k:{"uid":"b1e342ee-7f53-4dca-8959-8665aa74a26a"}:
+            .:
+            f:apiVersion:
+            f:blockOwnerDeletion:
+            f:controller:
+            f:kind:
+            f:name:
+            f:uid:
+      f:status:
+        .:
+        f:replicas:
+        f:selector:
+        f:unavailableReplicas:
+        f:updatedReplicas:
+    Manager:    manager
+    Operation:  Update
+    Time:       2020-05-22T19:06:05Z
+  Owner References:
+    API Version:           cluster.x-k8s.io/v1alpha3
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  Cluster
+    Name:                  eko
+    UID:                   b1e342ee-7f53-4dca-8959-8665aa74a26a
+  Resource Version:        5811363
+  Self Link:               /apis/controlplane.cluster.x-k8s.io/v1alpha3/namespaces/metal3/kubeadmcontrolplanes/eko
+  UID:                     e599039c-a72f-40a2-8c63-9126a1627527
+Spec:
+  Infrastructure Template:
+    API Version:  infrastructure.cluster.x-k8s.io/v1alpha4
+    Kind:         Metal3MachineTemplate
+    Name:         eko-controlplane
+    Namespace:    metal3
+  Kubeadm Config Spec:
```

Comment:
that's also quite some content, and it doesn't help that it's all formatted the same.

Maybe in this case it could be better to use `get -o yaml` instead of `describe` and format the code block as `YAML`? hopefully jekyll will then add syntax hilghlight to help understand all that content.

Even so, some of the content can also be `[... cut ...]`